### DISCUSSION TITLE
What is the size of 9x9 mesh

### DISCUSSION THREAD

**Discussion by vmachacek** [mriscoc/Ender3V2S1#1332] [Q&A] [answered]
What are physical dimensions of the mesh when I do 9x9 mesh and how far from edges is it located? I want to surgically pin point deficiencies as my mesh is all over the place

**Comment by mriscoc** [comment, marked as answer]
That depends on the mesh inset.

![mesh-inset](https://github.com/user-attachments/assets/270c9adf-adc2-4ee7-ab59-8dd7e94484dd)
